clients/cli/src/main/java/org/apache/gravitino/cli/MetalakeCommandHandler.java

```diff
@@ -40,15 +39,18 @@ public class MetalakeCommandHandler extends CommandHandler {
    * @param gravitinoCommandLine The Gravitino command line instance.
    * @param line The command line arguments.
    * @param command The command to execute.
-   * @param ignore Ignore server version mismatch.
+   * @param context The command context.
    */
   public MetalakeCommandHandler(
-      GravitinoCommandLine gravitinoCommandLine, CommandLine line, String command, boolean ignore) {
+      GravitinoCommandLine gravitinoCommandLine,
+      CommandLine line,
+      String command,
+      CommandContext context) {
     this.gravitinoCommandLine = gravitinoCommandLine;
     this.line = line;
     this.command = command;
-    this.ignore = ignore;
-    this.url = getUrl(line);
+    this.context = context;
+    this.context.setUrl(getUrl(line));
```

Comment:
The context object should be immutable. Is it possible to set the URL upon initialization?

Reply:
Currently, getUrl is part of CommandHandler, which is needed by the commands which have not been refactored yet. It might be possible to refactor after all commands have changed.

Reply:
OK, please refactor it later, as this will make the code clear.